check-pr-conventional-name/action.yml

```diff
@@ -1,9 +0,0 @@
-name: "Check if the pull request title follows conventional commits."
-description: "Action that checks if the title of the pull request is following the conventional commits standard."
-
-runs:
-  using: composite
-  steps:
-    - uses: amannn/action-semantic-pull-request@v5
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We do not inherit secrets. Instead, we are explicit when passing those as inputs. This is why the action was failing.

Reply:
Now, this is reading `${{ inputs.token }}`.